src/audit.ts

```diff
@@ -296,10 +335,10 @@ export default class Audit {
         for (const groupAudit of groupAudits) {
```

Comment:
Perhaps in the future we could also add an audit for last-reconfigure-request on the group.

Reply:
This has been done